Amazon.QLDB.Driver.Tests/TransactionExecutorTests.cs

```diff
@@ -21,11 +23,11 @@ public static void Setup(TestContext context)
         {
             mockResult = new Mock<IResult>();
             mockTransaction = new Mock<ITransaction>();
-            mockTransaction.Setup(transaction => transaction.Execute(It.IsAny<string>())).Returns(mockResult.Object);
-            mockTransaction.Setup(transaction => transaction.Execute(It.IsAny<string>(), It.IsAny<List<IIonValue>>()))
-                .Returns(mockResult.Object);
-            mockTransaction.Setup(transaction => transaction.Execute(It.IsAny<string>(), It.IsAny<IIonValue[]>()))
-                .Returns(mockResult.Object);
+            mockTransaction.Setup(transaction => transaction.Execute(It.IsAny<string>(), It.IsAny<CancellationToken>())).ReturnsAsync(mockResult.Object);
```

Comment:
In lambdas we usually go with the shorter `txn` rather than the longer `transaction`.